etc/base.yaml

```diff
@@ -251,6 +251,11 @@ radar_appserver:
   replicaCount: 1
   managementportal_resource_name: res_AppServer
   public_key_endpoints: []
+  smtp:
+    enabled: false # -- set to true, if sending of notifications via email should be enabled.
+    host: localhost
+    username: secret
```

Comment:
Username and password are in the base secrets so I think they shouldn't be here. Also rest of the SMTP settings change often so I think they should be here.

Reply:
I think I copied the situation from Management Portal. Should I clean that up as well?

Reply:
Yes thank you. These configs also are also missing in MP section of base.yaml

Reply:
Ok, I have cleanup up the SMTP settings for management portal and appserver. Is this ok like this?

Reply:
Yes!